packages/math-input/src/data/keys.ts

```diff
@@ -3,122 +3,70 @@
  * alphanumeric characters.
  */
 
-export type Key =
```

Comment:
Another instance where we had two types that were basically identical!  Now we have a single enum to represent the various keys we support.